tests/ui/const-generics/adt_const_params/unsized_field-1.stderr

```diff
@@ -7,7 +7,7 @@ LL |
 LL | struct A([u8]);
    |          ---- this field does not implement `ConstParamTy_`
    |
-note: the `ConstParamTy_` impl for `[u8]` requires that `unstable feature: `unsized_const_params``
+note: the `ConstParamTy_` impl for `[u8]` requires that `unstable feature `unsized_const_params` is enabled`
```

Comment:
this kinda results in something like:
"requires that `unstable feature `unsized const params` is enabled`"

note how the feature name isnt actually in the backticks. we could drop the backticks and word it as "feature(unsized_const_params) is enabled" which would avoid this problem and still flow naturally I think.

Reply:
we could also go the entire way and just print `ClauseKind::UnstableFeature` as `#![feature({symbol})]` so you'd get stuff like:

"requries that `#![feature(unsized_const_params)]`"
or
`fn foo where #![feature(unsized_const_params)]`
which also seems.... Kind of okay?